src/election_anomaly/munge_routines/__init__.py

```diff
@@ -204,7 +227,7 @@ def replace_raw_with_internal_ids(
         table_df[['Id',internal_name_column]],how='left',left_on=element,right_on=internal_name_column)
     row_df=row_df.drop([internal_name_column],axis=1)
     row_df.rename(columns={'Id':f'{element}_Id'},inplace=True)
-    return row_df
+    return [row_df, error]
```

Comment:
This is a minor point but we haven't been wrapping our compound return values into a list; instead we've just been doing something like `return row_df, error`. Then the calling function catches them as a pair rather than a list, like `row_df, err = replace_raw_with_internal_ids()`.

Reply:
I put the lists in because I couldn't figure out how else to do the typing on the output. Didn't try tuples, though, which would probably work.